src/fides/api/ctl/database/seed.py

```diff
@@ -436,14 +436,15 @@ async def load_samples(async_session: AsyncSession) -> None:
         sample_connections = load_sample_connections_from_project()
         with sync_session() as db_session:
             for connection in sample_connections:
+                assert connection.key
```

Comment:
```suggestion
                assert connection.key, "Connection Key not valid"
```

With `assert` I think it usually makes sense to add a little flavor text, even if it is mostly redundant